src/sidebar/components/notebook-view.js

```diff
@@ -0,0 +1,80 @@
+import { createElement } from 'preact';
+import { useEffect } from 'preact/hooks';
+import propTypes from 'prop-types';
+
+import { withServices } from '../util/service-context';
+import useRootThread from './hooks/use-root-thread';
+import useStore from '../store/use-store';
+
+import ThreadList from './thread-list';
+
+/**
+ * @typedef NotebookViewProps
+ * @prop {Object} [loadAnnotationsService] - Injected service
+ */
+
+/**
+ * The main content of the "notebook" route (https://hypothes.is/notebook)
+ *
+ * @param {NotebookViewProps} props
+ */
+function NotebookView({ loadAnnotationsService }) {
+  const focusedGroup = useStore(store => store.focusedGroup());
+  const setSortKey = useStore(store => store.setSortKey);
+
+  // Reload annotations if focused group changes
+  useEffect(() => {
+    // Load all annotations in the group, unless there are more than 5000
+    // of them: this is a performance safety valve
+    setSortKey('Newest');
+    if (focusedGroup) {
+      loadAnnotationsService.load({
+        groupId: focusedGroup.id,
+        maxResults: 5000,
+      });
+    }
+  }, [loadAnnotationsService, focusedGroup, setSortKey]);
+
+  const rootThread = useRootThread();
+  const groupName = focusedGroup?.name ?? '…';
+
+  const hasResults = rootThread.totalChildren > 0;
+  const threadCount =
+    rootThread.totalChildren === 1
+      ? '1 thread'
+      : `${rootThread.totalChildren} threads`;
```

Comment:
I'm not sure whether we want to use the "threads" terminology in the UI but I think that is a subject for a larger discussion in Slack and I suggest we don't let that get in the way of getting this merged.

Reply:
Yep! Discussion started in Slack.